playwright/test/workPrototype.test.ts

```diff
@@ -0,0 +1,81 @@
+import {
```

Comment:
I'm going to assume this isn't a prototype, but rather requesting items? Should we just call it that?

Reply:
It is a prototype in the sense it is the work we are about to test with users. However, it replicates the tests in work.test.ts, as the changes to the UI would cause those tests to fail. I envisage that once we're happy with the prototype, this file will get renamed and replace the current work.test.ts

Reply:
I'm just really sceptical of calling things prototypes, as everything so far has just been the first version of the thing, so why not just call it that.

Do you think that these tests would go in `work`? Would they not be their own suite?

Reply:
I think they make sense on work, they are about the information displayed on the work page. When we start the requesting stuff that would probably be a new test suite.

I'd like to keep the prototype name, as we're testing and iterating on it, once it's finalised I'll swap this for the work.test.ts file